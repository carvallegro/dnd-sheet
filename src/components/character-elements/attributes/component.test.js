import React from 'react'
import { shallow } from 'enzyme'

import Attributes from './index'

describe('<Attributes />', () => {
  it('renders', () => {
    const wrapper = shallow(<Attributes attributes={[]} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with an attribute', () => {
    const wrapper = shallow(
      <Attributes attributes={[{ name: 'TestAT', base: 12, modifier: 1 }]} />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
