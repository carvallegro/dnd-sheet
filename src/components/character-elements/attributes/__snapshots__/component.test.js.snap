// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Attributes /> renders 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Attributes /> renders with an attribute 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
